src/cfnlint/rules/resources/properties/Properties.py

```diff
@@ -177,10 +177,11 @@ def check_exceptions(self, parenttype, proptype, text):
             - Start with handling exceptions for templated code.
         """
         templated_exceptions = {
-            'AWS::ApiGateway::RestApi': ['BodyS3Location'],
+            'AWS::ApiGateway::RestApi': ['S3Location'],
```

Comment:
Isn't this [BodyS3Location](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-restapi.html#cfn-apigateway-restapi-bodys3location) or [BodyS3Location](https://github.com/aws/aws-cli/blob/master/awscli/customizations/cloudformation/artifact_exporter.py#L378)

Reply:
![Screen Shot 2020-09-08 at 10 54 22 AM](https://user-images.githubusercontent.com/7014355/92511083-b112c180-f1c1-11ea-8de5-804f419356e5.png)